composer.json

```diff
@@ -31,7 +31,11 @@
         ]
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true,
```

Comment:
When i run composer test, composer add theses lines ?

Reply:
This new configuration appeared with Composer 2.2. It's ok to add it.

May you move `allow-plugins` before `sort-packages`? It will make the build green again.